frontends/web/src/routes/account/receive/receive.tsx

```diff
@@ -284,20 +275,10 @@ export const Receive: FunctionComponent<Props> = ({
           </div>
         </div>
       </div>
-      <Guide>
-        <Entry key="guide.receive.address" entry={t('guide.receive.address')} />
-        <Entry key="guide.receive.whyVerify" entry={t('guide.receive.whyVerify')} />
-        <Entry key="guide.receive.howVerify" entry={t('guide.receive.howVerify')} />
-        <Entry key="guide.receive.plugout" entry={t('guide.receive.plugout')} />
-        {currentAddresses && (
-          <>
-            {currentAddresses.length > 1 && <Entry key="guide.receive.whyMany" entry={t('guide.receive.whyMany')} />}
-            {currentAddresses.length > 1 && <Entry key="guide.receive.why20" entry={t('guide.receive.why20')} />}
-            {currentAddresses.length > 1 && <Entry key="guide.receive.addressChange" entry={t('guide.receive.addressChange')} />}
-            {receiveAddresses && receiveAddresses.length > 1 && currentAddresses.length > 1 && <Entry key="guide.receive.addressFormats" entry={t('guide.receive.addressFormats')} />}
-          </>
-        )}
-      </Guide>
+      <ReceiveGuide
+        hasMultipleAddresses={currentAddresses && currentAddresses.length > 1}
+        hasDifferentFormats={receiveAddresses && receiveAddresses.length > 1}
+      />
     </div>
   );
 };
```

Comment:
maybe there could be other parts removed that are only relelvant to BB01 but I am not sure, cc @benma